apps/functions-subscription/src/system-env.ts

```diff
@@ -0,0 +1,14 @@
+import * as RTE from 'fp-ts/ReaderTaskEither';
+import * as TE from 'fp-ts/TaskEither';
+import { insertSubscription } from './use-cases/insert-subscription';
+
+type RemoveReaderType<T> = T extends (
```

Comment:
**suggestion (non-blocking)**: Adding a comment to explain why this type utility is needed will be helpful for us in the future

Reply:
✅